### PR TITLE
FIX(swiftlint) spacing issues and such

### PR DESCRIPTION
**Pull request by CMasterson**
## Summary
Remove offending whitespaces, parentheses and such.

### CODE REVIEW

**Review by Gio2018 — CHANGES_REQUESTED** [review]
Hey @CMasterson , I see a bunch of conflict markers that made into this PR, would you mind resolving those? Thank you!

**Comment by CMasterson** [comment]
> Hey @CMasterson , I see a bunch of conflict markers that made into this PR, would you mind resolving those? Thank you!

Silly mistake on my part. Should be resolved now.

File: PocketKit/Tests/SharedPocketKitTests/LegacyUserMigrationTests.swift [hunk 1]
```diff
@@ -235,6 +235,8 @@ extension LegacyUserMigrationTests {
             return try! JSONSerialization.data(withJSONObject: correct)
         }
 
+        let expectation = XCTestExpectation(description: "Migration event fired successfully.")
+
```
Comment (cyndichin) [inline PocketKit/Tests/SharedPocketKitTests/LegacyUserMigrationTests.swift:239]: doesn't seem like we are using this expectation, curious on why its being added.

File: PocketKit/Tests/PocketKitTests/Support/MockPocketBraze.swift [hunk 2]
```diff
@@ -46,4 +46,9 @@ extension MockPocketBraze {
 
         return calls[index] as? DidRecieveUserNotifcationImpl
     }
+
+    func signedInUserDidBeginMigration() {
+
+    }
```
Comment (cyndichin) [inline PocketKit/Tests/PocketKitTests/Support/MockPocketBraze.swift:52]: same as other comment

**Comment by CMasterson** [comment]
Linting issues are addressed by another PR

**Pull request closed without merging**